java-extras/build.gradle

```diff
@@ -1,5 +1,8 @@
 description = 'TripleA library for low-level helper APIs, ie: syntactic sugar'
 
 dependencies {
+    // https://mvnrepository.com/artifact/com.github.ben-manes.caffeine/caffeine
+    compile group: 'com.github.ben-manes.caffeine', name: 'caffeine', version: '2.8.1'
```

Comment:
We usually use the single string notation where the different parts are separated with a colon + we have all the version numbers in the main build.gradle file